app/assets/javascripts/miq_application.js

```diff
@@ -320,7 +320,10 @@ function miqCheckForChanges() {
     }
   }
   // use default browser reaction for onclick
-  return true;
+  if(obj && (href = obj.getAttribute("href")))
```

Comment:
@AparnaKarve this may be just me sleep-typing, but maybe this will catch all the url changes that are supposed to be changes?...

```js
	$stateProvider.state("otherwise", {
		url: "*path",
		controller: function($stateParams) {
			console.log('otherwise', $stateParams.path);  // same as $location.path()
		},
	})
```